pkg/ipamd/ipamd.go

```diff
@@ -194,6 +194,7 @@ type IPAMContext struct {
 	maxIPsPerENI         int
 	maxENI               int
 	unmanagedENI         int
+	cniunmanagedENI      int
```

Comment:
What's the use for `cniunmanagedENI` ?

Reply:
I need to update it here - https://github.com/aws/amazon-vpc-cni-k8s/blob/56851f0905dba4852eb895ec1c7bd5b1876a9c67/pkg/ipamd/ipamd.go#L637. Will update the PR.

Reply:
removed this since we the final value in unmanagedENI.